### PR TITLE
fix: okay-browser-daily

### PR DESCRIPTION
**Pull request by valekjo**
This PR updates `okay-browser-daily` Actor, so it can use product sitemaps instead of collections, and improves scraping on the detail page.

It works, but is super slow, so there is also alternative and faster solution that
 - uses product sitemaps to get product detail links
 - emulates what would happen with dom when the page was loaded
 - scrapes data from that

The emulation uses data from one additional endpoint, pre-fetched shopify config, and script copied form okay.

This is added as `okay-daily-v2`, needed to update yarn.lock for that.

See comments in PR for more details.

### CODE REVIEW

File: actors/okay-daily-v2/external-scripts/product-price-by-tags.js
Comment (valekjo) [inline actors/okay-daily-v2/external-scripts/product-price-by-tags.js:1]: This is a script that's just copied from okay webpage, without changes.

File: actors/okay-daily-v2/main.js [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+  const {
+    development,
+    debug,
+    proxyGroups,
+    maxRequestRetries,
+    country = Country.CZ,
+    customTableName = null,
+    type = ActorType.Full,
+    forceApifyProxy = false,
```
Comment (valekjo) [inline actors/okay-daily-v2/main.js]: This is new field on input.

**Comment by MartinaGelnerova** [comment]
@rarous pls merge 🙏

**Comment by rarous** [comment]
@valekjo Can you, please, make just one actor, that is the best solution and delete all others? I really don't like growing junk :)

**Comment by valekjo** [comment]
@rarous Ok, kept just one, named `okay`.

**Comment by rarous** [comment]
@valekjo can you please try to rebase on `trunk` and try to fix `yarn.lock` conflicts? Otherwise, I'm unable to merge your changes.

**Comment by valekjo** [comment]
@rarous Should work now.

**Comment by rarous** [comment]
Thank you ❤️

**Pull request merged**